javascript/javascript1/week2/WeatherWear.js

```diff
@@ -0,0 +1,15 @@
+function whatToWear(temperature) {
+  let message;
+  if (temperature >= 0 && temperature <= 15) {
+    message =
+      "The weather outside is cold and do not forget to wear warm clothes!";
```

Comment:
perfect. You can also return the string message directly at all conditions like
```js
if (temperature >= 0 && temperature <= 15) {
    return "The weather outside is cold and do not forget to wear warm clothes!";
```
You don't have to change the current code. Just I wanted to say that this is possible